packages/compass-query-bar/src/components/query-option/query-option.module.less

```diff
@@ -4,7 +4,6 @@
   display: flex;
   min-height: 28px;
   padding: 4px 5px;
-  border-top: 1px solid @gray7;
```

Comment:
There are now no borders between rows in the component as this `.query-option-toggle-row` that you introduced and moved the border to is only rendered for the first row:

<img width="813" alt="image" src="https://user-images.githubusercontent.com/5036933/133968641-460f6ea8-2487-4b8b-b43c-5761b33849e6.png">

If you want to remove it only for the first row, you would probably need to introduce some additional logic in the query bar / query option components so you can distinguish between them. A possible approach can be to add a special class name for this case and then you can use it to remove the border (or add it only when needed). Something like this (some pseudocode):

```css
.component {
  /* doesn't have border-top by default */
}

.component:not(.is-first) {
/*             ^ this classname needs to be somehow provided on the component first */
  border-top: 1px solid @gray7;
}
```